app/assets/stylesheets/screen/_header.scss

```diff
@@ -101,8 +101,11 @@ header {
       }
 
       ul.dropdown-menu {
-        top: 78%;
-        left: 24px;
+        top: auto;
+        left: auto;
+        margin-top: 5px;
+        text-align: start;
```

Comment:
Same comment as in other PR. Try `$left` or `$right`? And please add screenshots.